libs/api/domains/license-service/src/lib/licenseService.service.ts

```diff
@@ -474,4 +483,53 @@ export class LicenseServiceService {
     }
     return verification.data
   }
+
+  async createBarcode(
+    { nationalId }: User,
+    genericUserLicense: GenericUserLicense,
+  ) {
+    const code = randomUUID()
```

Comment:
The reason we use a short 10 letter unique id is to keep the jwt payload as small as possible. Else I would have used normal UUID